cosmo/monitors/acq_monitors.py

```diff
@@ -261,6 +261,9 @@ def plot(self):
         self.figure.add_traces(traces)
         self.figure['layout'].update(layout)
 
+    def store_results(self):
+        pass
```

Comment:
raise NotImplementedError or make abstract?

Reply:
It's supposed to be implemented, but we're waiting on these to be "finalized". In the meantime, the plots still needed to be generated

Reply:
Maybe... but maybe it would be better to throw and catch, I guess it depends on how many places this would need to catch?